srv/salt/ceph/stage/prep/master/default-no-update-no-reboot.sls

```diff
@@ -11,6 +10,11 @@ validate failed:
 
 {% endif %}
 
+crc_method:
+  salt.state:
+    - tgt: {{ master }}
+    - sls: ceph.salt.crc
```

Comment:
Any reason why `failhard: True` is omitted here?

Reply:
I think it shouldn't failhard here. It's more or less just suppressing a warning (at least that's the surfacing result when not applying this config option).

Reply:
If failure of this state doesn't cause the whole Stage to fail, then obviously there's no reason to add failhard.

I guess I have a testing mentality - whenever a program continues running after doing something that will cause the test to fail, it makes debugging the failure harder.